libvg/src/utf8.rs

```diff
@@ -0,0 +1,29 @@
+use byond::call::return_to_byond;
+use encoding::all::WINDOWS_1252;
+use encoding::label::encoding_from_windows_code_page;
+use encoding::types::DecoderTrap;
+use libc;
+use std::ffi::CStr;
+use std::slice;
+use std::ptr::null;
+
+#[no_mangle]
+pub extern "C" fn to_utf8(n: libc::c_int, v: *const *const libc::c_char) -> *const libc::c_char {
+    if n != 2 {
```

Comment:
@ComicIronic you are confusing args.

The function args here are not symmetrical with the args used when invoking in byond.

In this case byond encodes the args in an array, then it passes the length of the array and the array to the function.

This has the n arg because it's required to be there.